packages/interface/src/store/reducer.js

```diff
@@ -78,11 +78,51 @@ export function multipleEnableItems(
 	};
 }
 
+/**
+ * Reducer returning the user preferences.
+ *
+ * @param {Object}  state  Current state.
+ * @param {Object}  action Dispatched action.
+ *
+ * @return {Object} Updated state.
+ */
+export const preferences = flow( [ combineReducers ] )( {
+	featuresDefaults( state = {}, action ) {
+		if ( action.type === 'SET_FEATURE_DEFAULTS' ) {
```

Comment:
The question is whether we want to persist features defaults. I would say we don't want it because it's something that isn't a user setting.

On the other hand, the current proposal is closer to what existed before with the initial state that caused that default values would get persisted in the local storage causing some surprises, example:
https://github.com/WordPress/gutenberg/pull/19177#issuecomment-580633600
We want to persist user preferences also on the server via user meta, so that would mean that users that loaded default values for the first time, would have them stored as their lifetime choices.

Reply:
Good point! I hadn't thought of that side-effect.

I guess moving it to a separate non-persisted reducer would be the easiest fix.